x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -1062,9 +1064,10 @@ function trackAlertDurations<
     const state = originalAlertIds.includes(id)
       ? originalAlerts[id].getState()
       : currentAlerts[id].getState();
-    const duration = state.start
-      ? (new Date(currentTime).valueOf() - new Date(state.start as string).valueOf()) * 1000 * 1000 // nanoseconds
-      : undefined;
+    const durationInMs =
+      new Date(currentTime).valueOf() - new Date(state.start as string).valueOf();
+    const durationInNanoStr = durationInMs !== 0 ? `${durationInMs}000000` : '0';
```

Comment:
We are doing this conversion in multiple places, can we have a helper functions like `ms2ns` and `ns2ms`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/130819/commits/ea053d6186d740c8eda90ea2e60b5a585fc8357e and https://github.com/elastic/kibana/pull/130819/commits/082120b580bae1bca0f007e1ce382408588d504c.